### PR TITLE
Announcement for 1.4.

### PR DESCRIPTION
**Pull request by oaktowner**
This contains edits to the Announcement for Istio 1.4.

### CODE REVIEW

File: content/en/news/2019/announcing-1.4/_index.md [hunk 1]
```diff
@@ -11,4 +11,49 @@ aliases:
 
 We are pleased to announce the release of Istio 1.4!
 
+Version 1.4 continues our efforts at improving the user experience in Istio,
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.4/_index.md]:
```suggestion
Istio 1.4 continues our efforts at improving the Istio user experience,
```

File: content/en/news/2019/announcing-1.4/_index.md [hunk 2]
```diff
@@ -11,4 +11,49 @@ aliases:
...
+## Mixer-less telemetry
+Our implementation of telemetry without Mixer will simplify installation and
+operation of meshes, all while vastly improving performance. The in-proxy
+generation of HTTP metrics has graduated from experimental to alpha. Users
+are very excited about this improvement and we are working hard to get it
+ready. We've also added new experimental features that don't require Mixer:
+TCP metrics and Stackdriver metrics.
+
+## Improved troubleshooting
+The `istioctl analyze` command continues to get new functionality for
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.4/_index.md]: This is a new feature for 1.4.
```suggestion
We're introducing the [`istioctl analyze`](/docs/ops/diagnostic-tools/istioctl-analyze/) command to
```

File: content/en/news/2019/announcing-1.4/_index.md [hunk 3]
```diff
@@ -11,4 +11,49 @@ aliases:
...
+improved troubleshooting of your service mesh. Check for problems with
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.4/_index.md]:
```suggestion
improve troubleshooting of your mesh. Check for problems with
```

File: content/en/news/2019/announcing-1.4/_index.md [hunk 4]
```diff
@@ -11,4 +11,49 @@ aliases:
...
+configuration in your mesh, and even validate new configuration prior to
+submitting it to the mesh.
+
+## Better sidecar
+We've been doing tons of work to improve Envoy, its feature set, and the
+epxerience of using it. Envoy now exits more gracefully on crashes, supports
+more metrics, and can mirror traffic to a percentage of traffic. It reports
+the direction of traffic and has better configurability for stat patterns.
+Finally, there is a new experimental command that can tell you when
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.4/_index.md]:
```suggestion
Finally, there is a new [experimental command](/docs/reference/commands/istioctl/#istioctl-experimental-wait) that can tell you when
```

File: content/en/news/2019/announcing-1.4/_index.md [hunk 5]
```diff
@@ -11,4 +11,49 @@ aliases:
...
+We were very proud to be called out as one of the 
+[fastest growing](https://octoverse.github.com/#top-and-trending-projects)
+open source projects in all of Github. Want to get involved? Join one of our
+[Working Groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)
+and help us make Istio even better.
+
+To join the conversation, go to [discuss.istio.io](https://discuss.istio.io),
+log in with your GitHub credentials and join us!
+
 {{< relnote >}}
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.4/_index.md]: This needs to go at the top of the file, right after the first paragraph. This is the trigger to render the buttons in the UI.

**Comment by geeknoid** [comment]
And in case you miss it, there are a few lint errors:

   ./en/news/2019/announcing-1.4/_index.md
       34 | epxerience of using it. Envoy now exits  
       36 | ion of traffic and has better configurability for stat patterns. 
       52 | pen source projects in all of Github. Want to get involved? Join o 
>> 3 spelling errors found in 456 files
To learn how to address spelling errors, please see https://istio.io/about/contribute/creating-and-editing-pages/#linting
./en/news/2019/announcing-1.4/_index.md:50: MD009 Trailing spaces
./en/news/2019/announcing-1.4/_index.md:18: MD022 Headers should be surrounded by blank lines
./en/news/2019/announcing-1.4/_index.md:26: MD022 Headers should be surrounded by blank lines
./en/news/2019/announcing-1.4/_index.md:32: MD022 Headers should be surrounded by blank lines
./en/news/2019/announcing-1.4/_index.md:40: MD022 Headers should be surrounded by blank lines

File: content/en/news/2019/announcing-1.4/_index.md [hunk 6]
```diff
@@ -11,4 +11,49 @@ aliases:
...
+configuration in your mesh, and even validate new configuration prior to
+submitting it to the mesh.
+
+## Better sidecar
+We've been doing tons of work to improve Envoy, its feature set, and the
+epxerience of using it. Envoy now exits more gracefully on crashes, supports
+more metrics, and can mirror traffic to a percentage of traffic. It reports
+the direction of traffic and has better configurability for stat patterns.
+Finally, there is a new experimental command that can tell you when
+has been pushed to all proxies in the mesh.
```
Comment (howardjohn) [inline content/en/news/2019/announcing-1.4/_index.md]:
```
config has been pushed to all proxies in the mesh.
```

File: content/en/news/2019/announcing-1.4/_index.md [hunk 7]
```diff
@@ -11,4 +11,49 @@ aliases:
 
 We are pleased to announce the release of Istio 1.4!
 
+Version 1.4 continues our efforts at improving the user experience in Istio,
+with a focus on simplification. We've also continued to add features that
+improve the performance and experience of running Istio.
+
+## Mixer-less telemetry
+Our implementation of telemetry without Mixer will simplify installation and
+operation of meshes, all while vastly improving performance. The in-proxy
```
Comment (howardjohn) [inline content/en/news/2019/announcing-1.4/_index.md]: Do we want to be adding links to docs for these things so folks can get more details on these? Not sure if we prefer to just keep this high level and have the release notes handle that.

Comment (oaktowner) [reply content/en/news/2019/announcing-1.4/_index.md]: We didn't last time -- I just followed the convention set there. General descriptions (without links) of highlights at the top, followed by release notes with links.

File: content/en/news/2019/announcing-1.4/_index.md (hunk 6 above)
Comment (oaktowner) [reply content/en/news/2019/announcing-1.4/_index.md]: done

File: content/en/news/2019/announcing-1.4/_index.md (hunk 5 above)
Comment (oaktowner) [reply content/en/news/2019/announcing-1.4/_index.md]: done (and thanks for the lint errors -- still trying to get make serve work on my chromebook

File: content/en/news/2019/announcing-1.4/_index.md (hunk 4 above)
Comment (oaktowner) [reply content/en/news/2019/announcing-1.4/_index.md]: done

File: content/en/news/2019/announcing-1.4/_index.md (hunk 3 above)
Comment (oaktowner) [reply content/en/news/2019/announcing-1.4/_index.md]: done

File: content/en/news/2019/announcing-1.4/_index.md (hunk 2 above)
Comment (oaktowner) [reply content/en/news/2019/announcing-1.4/_index.md]: done. for some reason I thought it was in 1.3. Thanks.

File: content/en/news/2019/announcing-1.4/_index.md (hunk 1 above)
Comment (oaktowner) [reply content/en/news/2019/announcing-1.4/_index.md]: done

**Pull request merged**